### ISSUE TITLE
crash upon switch from pressure altitude to pressure

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#439]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/439) by Joern Ungermann (Bitbucket: [joernu76](https://bitbucket.org/joernu76), GitHub: [joernu76](https://github.com/joernu76)).**

----------------------------------------

```
2019-08-14 14:37:17,733 (_base._update_title_position:2515): update_title_pos
2019-08-14 14:37:17,818 (mss_qt.excepthook:172): MSS Version: 1.8.1
2019-08-14 14:37:17,818 (mss_qt.excepthook:173): Python Version: 3.7.3 (default, Mar 27 2019, 22:11:17)
[GCC 7.3.0]
2019-08-14 14:37:17,831 (mss_qt.excepthook:174): Platform: Linux-4.9.0-9-amd64-x86_64-with-debian-9.9 (('64bit', ''))
2019-08-14 14:37:17,831 (mss_qt.excepthook:175): Fatal error: Traceback (most recent call last):
  File "/home/icg173/anaconda3/envs/mss-1.8.1/lib/python3.7/site-packages/mslib/msui/sideview.py", line 319, in set_options
    self.getView().set_settings(settings)
  File "/home/icg173/anaconda3/envs/mss-1.8.1/lib/python3.7/site-packages/mslib/msui/mpl_qtwidget.py", line 772, in set_settings
    self.set_vertical_extent(*settings["vertical_extent"])
  File "/home/icg173/anaconda3/envs/mss-1.8.1/lib/python3.7/site-packages/mslib/msui/mpl_qtwidget.py", line 714, in set_vertical_extent
    self.setup_side_view()
  File "/home/icg173/anaconda3/envs/mss-1.8.1/lib/python3.7/site-packages/mslib/msui/mpl_qtwidget.py", line 660, in setup_side_view
    self.redraw_yaxis()
  File "/home/icg173/anaconda3/envs/mss-1.8.1/lib/python3.7/site-packages/mslib/msui/mpl_qtwidget.py", line 574, in redraw_yaxis
    major_ticks2 = np.arange(major_ticks[-1], self.p_top - 1, -label_distance // 10)
IndexError: index -1 is out of bounds for axis 0 with size 0
```

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Rework ylabels to remove crash for low pressures.

Fix issue #439

**Comment by ReimarBauer** [comment]
**Original comment by Joern Ungermann (Bitbucket: [joernu76](https://bitbucket.org/joernu76), GitHub: [joernu76](https://github.com/joernu76)).**

----------------------------------------

Same issue is present on the server side for providing tick marks on plots.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Fixed vertical labeling also on server side.

Fix issue #439
